app/services/idv/step.rb

```diff
@@ -3,7 +3,7 @@ module Idv
   class Step
     include VendorValidated
 
-    def initialize(analytics:, idv_form:, idv_session:, params:)
+    def initialize(analytics: nil, idv_form:, idv_session:, params:)
```

Comment:
I'm setting `nil` as the default value to make passing in analytics optional. Eventually, none of the Step classes will have analytics as an argument. I made this change to keep this PR small. Otherwise, if I removed the analytics argument altogether, I would have had to update all the other classes, but I only wanted to focus on one at a time.

If any other classes that still expect to receive an analytics argument don't, the specs will fail.